db/migrate/20240312150459_add_back_columns_to_adopter_foster_profiles.rb

```diff
@@ -0,0 +1,6 @@
+class AddBackColumnsToAdopterFosterProfiles < ActiveRecord::Migration[7.0]
```

Comment:
Do we need this migration? adopter_account_id is already on adopter_foster_profiles table. Then we have a migration above to remove both. 

I think it makes more sense to just remove the adopter_account_id from adopter_foster_profiles.

Reply:
agreed